tutorials/circumcision.md

```diff
@@ -0,0 +1,63 @@
+## Circumcision Tutorial
+
+Welcome to the circumcision module. This code deals with circumcision at birth and voluntary medical male circumcision (VMMC) intervention. The most relevant files are listed below:
+
+- `src/hivpy/circumcision.py` - The circumcision module.
+- `src/tests/test_circumcision.py` - Tests for the circumcision module.
+- `src/hivpy/data/circumcision.yaml` - Circumcision data and variables.
+- `src/hivpy/circumcision_data.py` - A class for storing data loaded from `circumcision.yaml`.
+
+If there are any circumcision-related variables you would like to change before running your simulation, please change them in `circumcision.yaml`.
+
+### Population Initialisation
+
+Birth circumcision is initialised when population data is first created. There are currently two methods for initialising birth circumcision:
+
+- `init_birth_circumcision_all` - Initialises circumcision at birth for the entire male population, both born and unborn. Circumcision dates for all born individuals are recorded as the date of the start of the simulation, but cirucmcision dates for unborn individuals are calculated by finding the dates at which each individual's age would be 0.25.
```

Comment:
We should mention here that this doesn't currently take into account COVID interruption